tests/ui/cfg/diagnostics-reexport.stderr

```diff
@@ -1,5 +1,17 @@
+error[E0432]: unresolved import `a::x`
+  --> $DIR/diagnostics-reexport.rs:12:9
+   |
+LL | pub use a::x;
+   |         ^^^^ no `x` in `a`
+   |
+note: found an item that was configured out
```

Comment:
Not sure how difficult would it be to get it to say x has been configured out.

Reply:
It had listed below